pkg/apiserver/apiserver.go

```diff
@@ -268,8 +268,6 @@ func (g *APIGroupVersion) InstallREST(container *restful.Container, root string,
 
 	// TODO: port the rest of these. Sadly, if we don't, we'll have inconsistent
 	// API behavior, as well as lack of documentation
-	mux := container.ServeMux
```

Comment:
Yes, what we'd really want is to inherit from ServeMux, but net/http doesn't define an interface.